### ISSUE TITLE
Unknown option nr in TEST

### ISSUE THREAD

**Issue by machawk1** [oduwsdl/warrick#15]
When running test after initially installing Warrick, the script report:
```
Arguments: -D MAKEFILE -o MAKEFILE_LOGFILE.log -xc -nr -dr 2007-08-02 -T -nv http://www.cs.odu.edu/

Unknown option: nr
```
Perhaps this argument is being captured by the perl environment instead of the perl script, i.e., an option to consider when executing the script.

Issues derived from the command:
```
perl warrick.pl -D MAKEFILE -o MAKEFILE_LOGFILE.log -xc -nr -dr 2007-08-02 -T -nv http://www.cs.odu.edu/
```

This may prevent the output files from being generated, which may explain why the TEST script fails on the first check.

Perl v5.18.2
macOS 10.13.4

**Comment by machawk1** [comment]
This may actually be caused by the CPAN mirrors failing in the install process.

**Comment by machawk1** [comment]
Resolving the obsolete mirror option and verifying the dependencies are correctly installed still resolved of the original error reported.

**Comment by machawk1** [comment]
The `nr` option is not included in the param list at https://github.com/oduwsdl/warrick/blob/master/warrick.pl#L3357-L3423 for some reason. Adding it back in per the definition lower in the script allows Warrick to run without reporting the invalid param error.

**Comment by machawk1** [comment]
Closed in PR #16
